backend-shared/config/db/operations_test.go

```diff
@@ -1,264 +1,140 @@
-package db
+package db_test
 
 import (
 	"context"
-	"strings"
-	"testing"
 	"time"
 
-	"github.com/stretchr/testify/assert"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	db "github.com/redhat-appstudio/managed-gitops/backend-shared/config/db"
 )
 
 var timestamp = time.Date(2022, time.March, 11, 12, 3, 49, 514935000, time.UTC)
-var DefaultValue = 101
-
-func TestGetOperationById(t *testing.T) {
-	SetupforTestingDB(t)
-	defer TestTeardown(t)
-	dbq, err := NewUnsafePostgresDBQueries(true, true)
-	if !assert.NoError(t, err) {
-		return
-	}
-	defer dbq.CloseDatabase()
-
-	ctx := context.Background()
-
-	_, _, _, gitopsEngineInstance, _, err := CreateSampleData(dbq)
-	if !assert.NoError(t, err) {
-		return
-	}
-	operationput := Operation{
-		Operation_id:            "test-operation-1",
-		Instance_id:             gitopsEngineInstance.Gitopsengineinstance_id,
-		Resource_id:             "test-fake-resource-id",
-		Resource_type:           "GitopsEngineInstance",
-		State:                   OperationState_Waiting,
-		Operation_owner_user_id: testClusterUser.Clusteruser_id,
-	}
-
-	err = dbq.CreateOperation(ctx, &operationput, operationput.Operation_owner_user_id)
-	if !assert.NoError(t, err) {
-		return
-	}
-
-	operationget := Operation{
-		Operation_id: operationput.Operation_id,
-	}
-
-	err = dbq.GetOperationById(ctx, &operationget)
-	if !assert.NoError(t, err) {
-		return
-	}
-
-	assert.IsType(t, timestamp, operationget.Created_on)
-	assert.IsType(t, timestamp, operationget.Last_state_update)
-	operationget.Created_on = operationput.Created_on
-	operationget.Last_state_update = operationput.Last_state_update
-	assert.Equal(t, operationput, operationget)
-
-	operationNotExist := Operation{
```

Comment:
As mentioned above, I recommend preserving this 'inexistent primary key' test and moving it over to the Ginkgo version.